### PR TITLE
started on devise form styling, create functionality working

### PR DESCRIPTION
**Pull request by AFerg15**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -73,4 +73,24 @@ header {
 }
 .fa-icon {
   padding-right: 10px;
-}
\ No newline at end of file
+}
+
+//---FORM---//
+.form.form{
```
Comment (WalkerTexas-coder) [inline app/assets/stylesheets/application.scss:79]: this works?

**Review by WalkerTexas-coder — COMMENTED** [review]
is there a reason a lot of devise pages are being brought it right now?

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (AFerg15) [reply app/assets/stylesheets/application.scss:79]: It does...it took a lot of inspecting to find the class to target this.

**Review by WalkerTexas-coder — APPROVED** [review]
This is great!

**Pull request merged**